dev_scripts/release-container-image.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+"""
+Script to automate the release of a new Dangerzone container image.
+
+This script automates the manual steps described in the container image release
+process, including:
+1. Picking a release candidate image
+2. Attesting provenance and reproducibility
+3. Signing and publishing the image
+"""
+
+import json
+import re
+import subprocess
+import sys
+from pathlib import Path
+
+import click
+
+PROJECT_ROOT = Path(__file__).parent.parent
+IMAGE_NAME_PATH = PROJECT_ROOT / "share" / "image-name.txt"
+DOCKER_ENV_PATH = PROJECT_ROOT / "Dockerfile.env"
+
+
+def extract_debian_archive_date():
+    with DOCKER_ENV_PATH.open() as f:
+        content = f.read()
+
+    match = re.search(r"DEBIAN_ARCHIVE_DATE=(\d+)", content)
+    if match:
+        return match.group(1)
+    else:
+        return None
+
+
+def run(cmd, check=True, capture_output=False):
+    """Run a command and optionally capture output."""
+    click.echo(f"Running: {' '.join(cmd)}")
+    result = subprocess.run(cmd, check=check, capture_output=capture_output, text=True)
+    return result
+
+
+@click.command()
+@click.option("--commit", default=None)
```

Comment:
Let's add a description here, and in the rest of the CLI options, to show how they will be used. We can say for instance that by default we will use the HEAD of the Git branch.

Reply:
Also, specifically for this CLI option, we need to verify that the user passes the full SHA1 commit. Else, our `attest-provenance` script will fail.